### PR TITLE
Added menu drop down when logged in for profile navigation or logout

### PR DESCRIPTION
**Pull request by monotkate**
@YashdalfTheGray @Mctalian @LongArmMcGee

### CODE REVIEW

**Comment by monotkate** [comment]
I maaaay have broken logout tests? >.>

File: src/users/accountView/accountView.tpl.html [hunk 1]
```diff
@@ -8,5 +8,4 @@
         <h2><center>{{user.name}} (<i>{{user.username}}</i>)</center></h2>
         <h4><center>{{user.email}}</center><h4>
     </div>
-    <md-button class="md-raised md-warn" ng-click="logout()" flex>Logout</md-button>
```
Comment (Mctalian) [inline src/users/accountView/accountView.tpl.html]: Is there any harm in keeping the logout button on the account page as well as in the new menu? Just wondering.

If we opt to remove it like you're suggesting, you can delete the whole `describe('$scope.logout', ...)` section in `accountView.spec.js`.

**Comment by Mctalian** [comment]
Other than that comment above, I think it looks good. Maybe add icons to each of the items in the drop down? Might look tacky, not sure.

**Comment by YashdalfTheGray** [comment]
I think it would look tacky. Do text only.

Yash Kulshrestha
yash.kulshrestha@gmail.com
(614)563-1163

File: src/users/accountView/accountView.tpl.html (hunk 1 above)
Comment (monotkate) [inline src/users/accountView/accountView.tpl.html]: I could leave it on the account page.  No harm, I was just figuring they could still logout from the drop down while on the account page.

Comment (YashdalfTheGray) [inline src/users/accountView/accountView.tpl.html]: Yeah, do that.

**Comment by monotkate** [comment]
Updated putting the login button back on the account page - unbreaking tests :)

Good to go? @YashdalfTheGray @Mctalian

**Comment by YashdalfTheGray** [comment]
Yeah, should be good to go.

**Comment by Mctalian** [comment]
Approved! :+1:

**Pull request merged**